packages/core/src/errors-list.ts

```diff
@@ -0,0 +1,65 @@
+export interface ErrorDescriptor {
+  code: string;
```

Comment:
Because the error code number is not easy to understand.

Reply:
It's not code category. It is the string form of error code.

Reply:
How is the error to be matched in the program?

I see the custom error is exposed as a uniform `KuaiError` class so `instanceof` doesn't work here. Then we should use the `code` to match, if so, it's better to expose `code` as an enumeration instead of plain strings.

Reply:
Confirmed that these errors are designed as a uniform error and users don't need to know the details

Reply:
Maybe it should be modified as follows:
Remove the filed `_isKuaiError` of `KuaiError`, and modify the `isKuaiError()` to 
```
public static isKuaiError(other: unknown): other is KuaiError {
    return other instanceof KuaiError;
 }
```